catroid/src/main/res/values/arrays.xml

```diff
@@ -203,5 +203,57 @@
         <item>@string/brick_set_physics_object_type_no</item>
     </string-array>
 
-
+    <!--Multilingual activity  -->
+    <string-array name="Languages_items" translatable="false">
```

Comment:
do you mean `getDisplayName (Locale locale)`??
https://developer.android.com/reference/java/util/Locale.html#getDisplayName(java.util.Locale)